### DISCUSSION TITLE
Zeek Intelligence Framework issue

### DISCUSSION THREAD

**Discussion by wildzealot** [Security-Onion-Solutions/securityonion#3177] [Unsupported Versions] [answered]
I have added a collection of flat files, with the correct zeek intel framework format, to a folder called "opt/zeek/intel". I then added the appropriate redef line to the local.zeek file. For good measure, I also added them to the _load_.zeek file. When I run the state.hightstate command I get an error. I then run the so-zeek-restart anyway. After this, all of the lines I added pointing to my intel files have been removed from both the local.zeek and "_load_.zeek" files. I assume this is either due to Salt or Docker. Can anyone point me in the right direction? The documentation is lacking.

**Comment by TOoSmOotH** [comment, marked as answer]
https://github.com/Security-Onion-Solutions/securityonion/discussions/3178
